docs/source/_topic_map.yml

```diff
@@ -14,8 +14,8 @@ Topics:
         File: installing
       - Name: Quickstart
         File: quickstart
-      - Name: Installing Docker Machine drivers
-        File: docker-machine-drivers
+      - Name: Setting up the hypervisor drivers
```

Comment:
When I said projects I meant https://github.com/dhiltgen/docker-machine-kvm as @gbraad mentioned.   

> I'd rather align more with Minikube,

+1. So I take back what I said earlier and  +1 for `Driver plugin`